cmd/pinniped/cmd/kubeconfig_test.go

```diff
@@ -321,6 +321,7 @@ func TestGetKubeconfig(t *testing.T) {
         		      - --token=test-token
         		      command: '.../path/to/pinniped'
         		      env: []
+        		      provideClusterInfo: false
```

Comment:
We do want to set this to `true`, since we will eventually want to use that `KUBERNETES_EXEC_INFO` information in our exec plugin. I think we will want our exec plugin to be able to know what cluster it is talking to, which is included in the `KUBERNETES_EXEC_INFO`.